src/test/rgw/rgw_multi/tests.py

```diff
@@ -825,3 +851,81 @@ def test_set_bucket_policy():
     for _, bucket in zone_bucket:
         bucket.set_policy(policy)
         assert(bucket.get_policy() == policy)
+
+def test_bucket_sync_disable():
+    zonegroup = realm.master_zonegroup()
+    zonegroup_conns = ZonegroupConns(zonegroup)
+    buckets, zone_bucket = create_bucket_per_zone(zonegroup_conns)
+
+    for bucket_name in buckets:
+        disable_bucket_sync(realm.meta_master_zone(), bucket_name)
+
+    for bucket_name in buckets:
+        for zone in zonegroup.zones:
+            check_bucket_sync_status_obj_not_exist(zone, bucket_name)
```

Comment:
it's probably better to just loop over zones here, and grep the `log list` output for all bucket names at the same time